docassemble/AssemblyLine/data/questions/ql_baseline.yml

```diff
@@ -815,31 +815,17 @@ validation code: |
 # Customize this question if you know whose children they are in your interview!
 # this is just a very generic way to ask.
 id: children names without birthdate
-sets:
-  - children[0].name.first
-  - children[0].name.last
-  - children[0].name.middle
-  - children[0].name.suffix
-question: |
-  % if hasattr(children, 'ask_number') and children.ask_number and children.target_number == 1:
-  Child's name
-  % else:
-  Name of first child
-  % endif
-fields:
-  - code: |
-      children[0].name_fields()
----
-# Customize this question if you know whose children they are in your interview!
-# this is just a very generic way to ask.
-id: children names without birthdate
 sets:
   - children[i].name.first
   - children[i].name.last
   - children[i].name.middle
   - children[i].name.suffix
 question: |
+  % if hasattr(children, 'ask_number') and children.ask_number and children.target_number == 1:
```

Comment:
```suggestion
  % if hasattr(children, 'ask_number') and children.ask_number and children.target_number == 1 and i == 0:
```

Reply:
I'm happy to make the change, but I just want to check on my understanding. If the target number is 1, won't `i == 0` always resolve to `True`?

Reply:
no. target number is only used when you use .gather(). If there's an "add" button you can get past it.